server/build.sbt

```diff
@@ -23,8 +23,8 @@ libraryDependencies := Seq(
   "io.7mind.izumi"                %% "idealingua-v1-compiler"           % "0.8.6",
   "io.7mind.izumi"                %% "idealingua-v1-compiler"           % "0.8.6",
   "io.7mind.izumi"                %% "idealingua-v1-runtime-rpc-http4s" % "0.8.6",
-  "dev.zio"                       %% "zio"                              % "1.0.0-RC10-1",
```

Comment:
You can update to izumi `0.8.7` and bump zio / http4s versions back up to latest

Reply:
Ah, cool. I will update to `0.8.7`